binderhub/templates/index.html

```diff
@@ -131,7 +131,7 @@ <h3 class="text-center">How it works</h3>
         <span class="point point-red">2</span>
       </div>
       <div class="col-md-7 front">
-        <span class="front-em">We build a Docker image of your repository</span><br />Does your repository depend on files being loaded for your live notebooks? We will search in your repository's root for a dependency such as a requirements.txt, environment.yml or Dockerfile. The first dependency file found will be used to build a Docker image. If an image has already been built for the given repository, it will not be rebuilt. If a new commit has been made, the image will automatically be rebuilt.
+        <span class="front-em">We build a Docker image of your repository</span><br />Does your repository depend on files being loaded for your live notebooks? We will search in your repository's root for a dependency such as a requirements.txt or environment.yml (<a href="#">more details</a>). The first dependency file found will be used to build a Docker image. If an image has already been built for the given repository, it will not be rebuilt. If a new commit has been made, the image will automatically be rebuilt.
```

Comment:
we should change this wording..."the first dependency file found..." is not correct! Maybe say

"We will search in your repository's root for dependency files such as a requirements.txt or environment.yml (<a href="#">more details</a>). These files will be used to build a Docker image."

Reply:
Instead of:

> We will search in your repository's root for a dependency such as a requirements.txt or environment.yml (<a href="#">more details</a>). The first dependency file found will be used to build a Docker image. 

Binder will search for a dependency file, such as requirements.txt or environment.yml, in the repository's root directory (more details on more complex dependencies in documentation). The dependency file will be used to build a Docker image.

Reply:
Used @willingc's version with file -> fileS in the last sentence